src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -765,6 +765,16 @@ private void validateObservations(ValidationErrors validationErrors,
             // had been saved prior to import
             } else if (existingObsByObsHash.containsKey(importHash) && !isObservationMatched(importHash, importObsValue, phenoCol, rowNum)) {
 
+                // different data means validations still need to happen
+                // TODO consider moving these two calls into a separate method since called twice together
```

Comment:
yes, since we have validation in two places in the method (one for the overwrite case and one for the non-overwrite case, it feels like it should be cleaned up in some manner)